dev-tools/mage/common.go

```diff
@@ -754,22 +754,29 @@ func VerifySHA256(file string, hash string) error {
 // CreateSHA512File computes the sha512 sum of the specified file the writes
 // a sidecar file containing the hash and filename.
 func CreateSHA512File(file string) error {
+	computedHash, err := GetSHA512Hash(file)
+	if err != nil {
+		return err
+	}
+	out := fmt.Sprintf("%v  %v", computedHash, filepath.Base(file))
+
+	//nolint:gosec // permissions are correct
+	return ioutil.WriteFile(file+".sha512", []byte(out), 0644)
```

Comment:
```suggestion
	return os.WriteFile(file+".sha512", []byte(out), 0644)
```
ps https://pkg.go.dev/io/ioutil#WriteFile